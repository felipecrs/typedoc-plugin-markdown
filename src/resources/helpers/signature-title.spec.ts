import * as Handlebars from 'handlebars';

import { setup } from '../../../test/utils';

describe(`[helpers] signatureTitle`, () => {
  let project: any;

  beforeAll(() => {
    ({ project } = setup());
  });

  test(`should compile`, () => {
    const data = project.findReflectionByName('functionWithParameters');
    const result = Handlebars.helpers.signatureTitle.call(data.signatures[0]);
    expect(result).toMatchSnapshot();
  });
});
