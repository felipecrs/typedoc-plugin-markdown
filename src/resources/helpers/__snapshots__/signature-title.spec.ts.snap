// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[helpers] signatureTitle should compile 1`] = `
"**functionWithParameters**(\`paramZ\`: string, \`paramG\`: any, \`paramA\`: any): *number*
"
`;
